Source/items.cpp

```diff
@@ -2668,10 +2670,10 @@ int CheckUnique(int i, int lvl, int uper, bool recreate)
 		}
 	}
 
-	return idata;
+	return (_unique_items)idata;

```

Comment:
Instead of casting, can we type `idata` to `_unique_items` instead?

Reply:
see previous reply